<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Favorites</class>
 <widget class="QDialog" name="Favorites">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage favorites</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Static favorites</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Favorite contacts:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="favoriteContactsList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QPushButton" name="addContactButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/go-previous</normaloff>:/go-previous</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeContactButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/go-next</normaloff>:/go-next</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Available contacts:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="allContactsList"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Dynamic Favorites</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QRadioButton" name="noDynamicButtonButton">
        <property name="text">
         <string>Only show favorites from the list above.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showDynFavoritesButton">
        <property name="text">
         <string>Show contacts with the most messages.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="showDynFavoritesWithDaysButton">
          <property name="text">
           <string>Show contacts with the most messages within the last</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="dayBox">
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>days.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QCheckBox" name="submenuBox">
        <property name="text">
         <string>Show favorites in a sub-menu</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numBox">
          <property name="maximum">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>dynamic favorites</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>allContactsList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>addContactButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>favoriteContactsList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>removeContactButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Favorites</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Favorites</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDynFavoritesWithDaysButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>dayBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
